shared-bindings/synthio/LFO.c

```diff
@@ -57,7 +57,30 @@ STATIC const uint16_t triangle[] = {0, 32767, 0, -32767};
 //|
 //|     In the current implementation, LFOs are updated every 256 samples. This
 //|     should be considered an implementation detail, though it affects how LFOs
-//|     behave for instance when used to implement an integrator (``l.offset = l``).
+//|     behave for instance when used to implement an integrator (``l.offset = l``).A
+//|
+//|     An LFO's output is not updated in any other way than when its associated
+//|     synthesizer updates it. For instance, if an LFO is created and its "first"
+//|     output is non-zero, its output is 0 until it is updated by its associated synthesizer.
```

Comment:
I'm confused. did you mean if its first output is zero?

Reply:
Yeah, this is unclear.

A reasonable expectation is that for offset=0, gain=1, the LFO would start with its "value" property equal to waveform[0]. But it doesn't, the its .value property is 0 until it is updated by actually generating samples for some synthesizer.

Reply:
I'll re-word this but probably not tonight.